generate-templates.yaml

```diff
@@ -40,20 +40,22 @@
   - name: Load RHEL 8 versions
     set_fact:
       rhel8_labels: "{{ lookup('osinfo', 'distro=rhel') |map(attribute='short_id') |select('match', '^rhel8\\.') |list |sort }}"
-#
-#  - name: Generate RHEL 8 SAP HANA template
-#    template:
-#      src: rhel8.saphana.tpl.yaml
-#      dest: "{{ playbook_dir }}/dist/templates/{{ os }}-{{ item.workload }}-{{ item.flavor }}.yaml"
-#    with_items:
-#      - {flavor: tiny,  workload: saphana, default: False}
-#    vars:
-#      os: rhel8
-#      icon: rhel
-#      majorrelease: rhel8
-#      oslabels: "{{ rhel8_labels }}"
-#      osinfoname: "{{ oslabels[0] }}"
-#      cloudusername: cloud-user
+
+  # The SAP HANA template will not be generated
+  - name: Generate RHEL 8 SAP HANA template
+    when: 0 > 1 # This template is disabled
```

Comment:
why don't we delete this and then pull it back from git? Is this for some kind of visibility?

Reply:
Yes. See comment on previous PR: https://github.com/kubevirt/common-templates/pull/389#discussion_r767539174